enterprise_subsidy/apps/subsidy/migrations/0017_auto_20230905_1511.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 3.2.20 on 2023-09-05 15:11
```

Comment:
naming nit: we generally try to give the migration files a descriptive name when they're created, using the `-n` flag, e.g. `./manage.py makemigrations subsidy -n add_subsidy_is_hidden`.
This makes it easier to see at a glance which migrations have been applied which running `./manage.py showmigrations subsidy`.